fdbserver/SkipList.cpp

```diff
@@ -168,9 +167,15 @@ struct SortTask {
 };
 
 void sortPoints(std::vector<KeyInfo>& points) {
-	std::vector<SortTask> tasks;
-	std::vector<KeyInfo> newPoints;
-	std::vector<int> counts;
+	// Static locals are safer than thread_local in Flow's cooperative execution model
+	static std::vector<SortTask> tasks;
+	static std::vector<KeyInfo> newPoints;
+	static std::vector<int> counts;
```

Comment:
Using `static` is only fine with single-thread model of flow, which means the code is not portable for multi-threaded cases.